Lekce_04_Ukol/Program.cs

```diff
@@ -0,0 +1,129 @@
+            if (jeVSeznamu)
+            {
+                Console.WriteLine("Je v seznamu předmětů.");
+                Console.WriteLine("-----------------------");
+            }
+            else
+            {
+                Console.WriteLine("Není v seznamu předmětů.");
+                Console.WriteLine("-----------------------");
+            }
+
+            // 6. Vypiš do konzole, kolik je v tom listu prvků a
+            // připoj k tomu všechny ty hodnoty (např: "2: modra, zelena").
+            Console.WriteLine($"Počet prvků v listu je: {listPredmetu.Count()}");
+            Console.WriteLine("Seznam prvků: ");
+
+            VypisList(listPredmetu);
+            Console.WriteLine("-----------------------");
+
+            // 7. Vytvoř slovník, kde klíčem bude položka nákupu (string)
+            // a hodnotou cena té položky, a vlož nějaké hodnoty (např: <"chleba", 20>).
+            Dictionary<string, int> polozkyNakupu = new Dictionary<string, int>
+            {
+                { "chleba", 20 },
+                { "houska", 5 },
+                { "koláček", 15},
+                { "rohlík", 3},
+            };
+
+            VypisSlovnik(polozkyNakupu);
+            Console.WriteLine("-----------------------");
+
+            static void VypisSlovnik(Dictionary<string, int> polozkyNakupu)
+            {
+                foreach (var (pecivo, cena) in polozkyNakupu)
+                {
+                    Console.WriteLine($"{pecivo} : {cena}");
+                }
+            }
+
+            // 8. Zjisti, jestli slovník obsahuje nějakou konkrétní potravinu
+            // a pokud ano, vypiš její cenu, pokud ne, vypiš, že není.
+            VratCenuPolozky("rohlík", polozkyNakupu);
+            VratCenuPolozky("kukuřice", polozkyNakupu);
+            Console.WriteLine("-----------------------");
+
+            static void VratCenuPolozky(string pecivo, Dictionary<string, int> polozkyNakupu)
+            {
+                bool nasloSe = polozkyNakupu.TryGetValue(pecivo, out int cena);
+                if (nasloSe)
+                {
+                    Console.WriteLine($"Položka se našla - {pecivo} : {cena}");
+                }
+                else
+                {
+                    Console.WriteLine("Položka se nenašla.");
+                }
+            }
+
+            // 9. Řekněme, že už jsi do slovníku přidala např. chleba a zjistila,
+            // že máš v nákupní tašce ještě jeden -> uprav hodnotu k tomu klíči tak,
+            // aby obsahovala hromadnou cenu za všechny stejné položky.
+            VypisSlovnik(polozkyNakupu);
+            Console.WriteLine("-----------------------");
+            AktualizaceHromadneCeny("rohlík", polozkyNakupu);
+            VypisSlovnik(polozkyNakupu);
+            Console.WriteLine("-----------------------");
+
+            static void AktualizaceHromadneCeny(string pecivo, Dictionary<string, int> polozkyNakupu)
```

Comment:
nit: Metody by vzdy mely byt nejake sloveso
```suggestion
            static void AktualizujHromadneCeny(string pecivo, Dictionary<string, int> polozkyNakupu)
```